build-vm-zvm

```diff
@@ -223,7 +227,6 @@ zvm_worker_init() {
 }
 
 zvm_cp() {
-    modprobe vmcp || zvm_fatal "Cannod load vmcp module"
     if test -n "$1" ; then
```

Comment:
is manually loading vmcp no longer required at all ?

Reply:
Its actually not possible anymore with current kernel versions. Its not a module anymore, but compiled into the kernel. The reason why it is not in here at all is, that the obsworker script already tries to load this module (for older distributions) and loading the module repeatedly is not necessary at all.

Reply:
okay, that answered my question ... looks good to me then